stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -273,15 +296,15 @@ public final class _ExecutionContext {
 
     let ctx = TFE_NewContext(opts, status)
     checkOk(status)
-    self.cContext = ctx!
+    self.eagerCtx = ctx!
```

Comment:
`eagerCtx` -> `eagerContext`

Reply:
Changed, but from similar past discussion, i feel always spelling out the full word may not add additional clarity. ctx is a well-understood term.

Reply:
This is not a question of clarity, but a question of naming convention. I tried to grep `ctx` in `stdlib/core/` and I found none. Plus, the standard library implementation is consistently following the naming convention by spelling out words for type members, functions and types AFAIK. It's better that we strive to do the same.

Reply:
I still think this kind of convention is hard to maintain in an absolute way. Say we have a class named `MatrixMultiplicationManager` -- I think it would be clearer if it's instead sth like `MatmulMgr`. So where do we draw the line on when to stop using full words?

stdlib is an evolving entity. If someday it adds more systems-oriented libraries (e.g. for managing thread context, context switch, etc), I won't be surprised if some authors go for `ctx`.

It'd be better if we can relax our coding convention a bit on this subject, and leave it to personal choices (unless the chosen name reduces clarity).

Reply:
> Say we have a class named `MatrixMultiplicationManager` -- I think it would be clearer if it's instead sth like `MatmulMgr`.

"Manager" and "Context" are really common words across Swift code bases, and the conventional spelling of these two words _are_ the full form.

I have never seen anyone use "Mgr" in Swift on a type name. Plus, "Mgr" would _definitely not_ be considered acceptable as a _public_ API. If you are talking about internal API, these things are hard to argue about. In these case, I would choose not to argue but just refer to [Swift API Design Guidelines - Section "Use Terminology Well" - Item "Avoid abbreviations"](https://swift.org/documentation/api-design-guidelines).

>  If someday it adds more systems-oriented libraries (e.g. for managing thread context, context switch, etc), I won't be surprised if some authors go for `ctx`.

That would not surprise me either, only if it were local variables or private code.

I don't think the argument of "more system-oriented libraries" would make the case more convincing. For example, there's hardly anything more low-level than pointer types, but they are called `UnsafeMutableBufferPointer` instead of `UnsafeMutableBufPtr`. The reason for that is "Avoid abbreviations" defined in the Swift API design guidelines.

But again, if it's internal/private code, it's hard to argue about. However, in Swift readability reviews, we tend to treat external and internal code the same.